api/src/convert/block_metadata.rs

```diff
@@ -24,13 +29,31 @@ impl TryFrom<&blockchain::BlockMetadataContents> for BlockMetadataContents {
     fn try_from(src: &blockchain::BlockMetadataContents) -> Result<Self, Self::Error> {
         let block_id = src.get_block_id().try_into()?;
         let quorum_set = src.get_quorum_set().try_into()?;
-        let report = src.get_verification_report().try_into()?;
+        let attestation_evidence = match &src.attestation_evidence {
+            Some(evidence) => {
```

Comment:
Nit: You can merge the match statements and match on `Some(blockchain::BlockMetadataContents_oneof_attestation_evidence::dcap_evidence(evidence))` etc

Reply:
oh that would be better

Reply:
updated, note I brought in `BlockMetadataContents_oneof_attestation_evidence` to use directly as it allowed keeping the match expression on one line for cargo fmt